src/transformers/models/bloom/configuration_bloom.py

```diff
@@ -53,14 +53,16 @@ class BloomConfig(PretrainedConfig):
 
 
     Args:
-        vocab_size (`int`, *optional*, defaults to 50257):
+        vocab_size (`int`, *optional*, defaults to 250880):
             Vocabulary size of the Bloom model. Defines the number of different tokens that can be represented by the
```

Comment:
A small suggestion to highlight why there may be a difference between this number and the number of tokens known by the tokenizer (you will probably have to reformat the docstring if you are happy with this change)
```suggestion
            Vocabulary size of the Bloom model. Defines the maximum number of different tokens that can be represented by the
```

Reply:
Perfect thanks a lot!